### DISCUSSION TITLE
manually wire editing and move instance in OpenRoad

### DISCUSSION THREAD

**Discussion by louis-ovt** [The-OpenROAD-Project/OpenROAD#5627] [Q&A]
Hi experts

The question come across for the situation where I need to fine tune my floorplan which just requires  minor manual effort so 
Is there a way I can do manually wire editing and move instances like srams in openroad pnr tool?

Thanks
Louis

**Comment by maliberty** [comment]
OpenROAD is not well suited to this currently.  If you are working in gds you might use klayout or magic.

**Reply by louis-ovt** [reply]
okay, thank you for letting me know.

**Reply by louis-ovt** [reply]
thank you!
